mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -345,34 +364,48 @@ def _compute_moc_climo_postprocess(self):  # {{{
 
         config = self.config
 
-        dvEdge, areaCell, refBottomDepth, latCell, nVertLevels, \
-            refTopDepth, refLayerThickness = self._load_mesh()
-
         self.regionNames = config.getExpression(self.sectionName,
                                                 'regionNames')
 
         # Load basin region related variables and save them to dictionary
+        mpasMeshName = config.get('input', 'mpasMeshName')
+        regionMaskDirectory = config.get('regions', 'regionMaskDirectory')
+
+        regionMaskFile = '{}/{}_MOCBasinsAndTransectMasks.nc'.format(
```

Comment:
Here is where the region mask file gets named automatically.  We can change the name if you prefer something else.